jmx-metrics/src/integrationTest/java/io/opentelemetry/contrib/jmxmetrics/AbstractIntegrationTest.java

```diff
@@ -112,8 +112,8 @@ void beforeEach() {
     otlpServer.reset();
   }
 
-  @SafeVarargs
-  protected final void waitAndAssertMetrics(Consumer<Metric>... assertions) {
+  @SuppressWarnings("varargs")
```

Comment:
Don't think I see a varargs usage in this method

Reply:
Correct, fixed.